### PR TITLE
Fix environment colors on Environments Page

### PR DESCRIPTION
**Pull request by tameremad**
image

### CODE REVIEW

**Comment by tameremad** [comment]
https://github.com/freiheit-com/kuberpult/issues/442

File: services/frontend-service/src/ui/components/EnvironmentCard/EnvironmentCard.tsx [hunk 1]
```diff
@@ -62,6 +64,8 @@ export const EnvironmentCard: React.FC<{ environment: string }> = (props) => {
 
 export const EnvironmentGroupCard: React.FC<{ environmentGroup: EnvironmentGroup }> = (props) => {
     const { environmentGroup } = props;
+    // all envs in the same group have the same priority
+    const priorityClassName = useEnvironmentPriorityClassName(environmentGroup.environments[0]?.name);
```
Comment (leandro-freiheit) [inline services/frontend-service/src/ui/components/EnvironmentCard/EnvironmentCard.tsx:68]: This does not seem completely right. If the first element in `environmentGroup.environments` has a lower priority but the next element has a higher priority, then the priority of the envGroup will be lower than I expected. Shouldn't we look into every environment in the environment group and have the `priorityClassName` be equal to the highest priority environment in the environment group?

Comment (leandro-freiheit) [reply services/frontend-service/src/ui/components/EnvironmentCard/EnvironmentCard.tsx:68]: Or do we already sort the environments by priority?

File: services/frontend-service/src/ui/components/EnvironmentCard/EnvironmentCard.tsx [hunk 2]
```diff
@@ -13,16 +13,18 @@ You should have received a copy of the MIT License
...
 import { Locks } from '../../../images';
 import * as React from 'react';
 import { EnvironmentLockDisplay } from '../EnvironmentLockDisplay/EnvironmentLockDisplay';
 import { EnvironmentGroup } from '../../../api/api';
+import classNames from 'classnames';
 
 export const EnvironmentCard: React.FC<{ environment: string }> = (props) => {
     const { environment } = props;
     const locks = useFilteredEnvironmentLockIDs(environment);
+    const priorityClassName = useEnvironmentPriorityClassName(environment);
```
Comment (sven-urbanski-freiheit-com) [inline services/frontend-service/src/ui/components/EnvironmentCard/EnvironmentCard.tsx:27]: `useEnvironmentPriorityClassName` should not be a hook.
Suggestion: Make a `useEnvironment` Function
```
const useEnvironment = (string)=>Environment ...
```
Then write another function to get the priorityClassname, e.g.
```
const getPriorityClassName = (env:Environment):string => ...
```

File: services/frontend-service/src/ui/components/EnvironmentCard/EnvironmentCard.tsx (hunk 1 above)
Comment (tameremad) [reply services/frontend-service/src/ui/components/EnvironmentCard/EnvironmentCard.tsx]: https://github.com/freiheit-com/kuberpult/pull/683

**Review by leandro-freiheit — APPROVED** [review]

**Pull request merged**